doc/source/dev/api-dev/array_api.rst

```diff
@@ -121,6 +121,32 @@ Support is provided in `scipy.stats` for the following functions:
 `scipy.stats.jarque_bera`, `scipy.stats.bartlett`, `scipy.stats.power_divergence`,
 and `scipy.stats.monte_carlo_test`.
 
+Some features provide a capability table in the documentation like this:
+
++---------+-------------+-------------+--------------+
+| Library | CPU         | GPU         | ``marray``   |
++=========+=============+=============+==============+
+| NumPy   | ✓           | ✗           | ✓            |
```

Comment:
minor: probably better to use `x` to indicate support is missing when that's possible, and "n/a" when the cell can never be checked - NumPy on GPU or CuPy on CPU will never be a thing.